src/main/java/com/chessmates/utility/LichessApiImpl.groovy

```diff
@@ -78,6 +77,7 @@ class LichessApiImpl implements LichessApi {
 
         def pageResults = resultObjects.stream()
                 .map(parse)
+                .filter {item -> item != null }
```

Comment:
The other option is to use Optional of Game https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html